src/avfoundation.cs

```diff
@@ -9078,13 +9119,169 @@ interface AVPlayerItemMetadataOutputPushDelegate : AVPlayerItemOutputPushDelegat
 		void DidOutputTimedMetadataGroups (AVPlayerItemMetadataOutput output, AVTimedMetadataGroup [] groups, AVPlayerItemTrack track);
 	}
 
+	[Static]
+	interface AVVideoColorPrimaries {
+		[iOS (10, 0)]
+		[Field ("AVVideoColorPrimaries_ITU_R_709_2")]
+		NSString ITU_R_709_2 { get; }
+
+		[NoiOS]
+		[Field ("AVVideoColorPrimaries_EBU_3213")]
+		NSString EBU_3213 { get; }
+
+		[iOS (10, 0)]
+		[Field ("AVVideoColorPrimaries_SMPTE_C")]
+		NSString SMPTE_C { get; }
```

Comment:
3+ letters acronyms -> `Smpte`

Make sure to run intro tests for possible typos to whitelist.